easybuild/easyconfigs/s/SAMtools/SAMtools-0.1.20-foss-2015b.eb

```diff
@@ -0,0 +1,29 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#
+# Copyright:: Copyright 2012-2014 Uni.Lu/LCSB, NTUA
+# Authors::   Robert Schmidt <roschmidt@ohri.ca>, Cedric Laczny <cedric.laczny@uni.lu>, Fotis Georgatos <fotis@cern.ch>
+# License::   MIT/GPL
+# $Id$
+#
+##
+
+name = 'SAMtools'
+version = '0.1.20'
+
+homepage = 'http://www.htslib.org/'
+description = """SAM Tools provide various utilities for manipulating alignments in the SAM format,
+ including sorting, merging, indexing and generating alignments in a per-position format."""
+
+toolchain = {'name': 'foss', 'version': '2015b'}
+toolchainopts = {'optarch': True, 'pic': True}
+
+source_urls = ['https://github.com/samtools/samtools/archive']
+sources = ['%(version)s.tar.gz']
```

Comment:
this is slightly different from existing `SAMtools` 0.1.20 easyconfigs, any particular reason?

also, the existing easyconfigs include a patch related to `ncurses`, why not here?